argocd.rst

```diff
@@ -0,0 +1,82 @@
+ArgoCD Configuration
+====================
+
```

Comment:
Patrick and I are adding a `note` at the beginning as a placeholder for an abstract. May be a good idea to add something similar here.